### PR TITLE
Add database schema and test data.

### PR DESCRIPTION
**Pull request by magical**
CO-401

Generated by SQL Developer.

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+--------------------------------------------------------
+--  DDL for Table MUS_STATUS
+--------------------------------------------------------
+
+  CREATE TABLE "MUS_STATUS" 
+   (	"ID" NUMBER, 
+	"NAME" VARCHAR2(100)
+   ) ;
+REM INSERTING into MUS_ALBUM
+SET DEFINE OFF;
```
Comment (floppyears) [inline schema.sql]: Let's separate the test data from the DDL, constraints and trigger. Just create a separate file where you keep the test data.

File: schema.sql [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+Insert into MUS_ALBUM (ID,ARTIST_ID,TITLE,EDITION,RELEASED,CREATED,STATUS) values (1,1,'Dark Side of the Moon','30th Anniversary SACD',to_date('03-MAR-71','DD-MON-RR'),to_date('24-MAY-16','DD-MON-RR'),2);
```
Comment (floppyears) [inline schema.sql]: 👍

**Comment by magical** [comment]
@jose Done. Please take another look.

File: schema.sql (hunk 2 above)
Comment (magical) [inline schema.sql]: :new_moon: :rainbow:

**Pull request merged**